src/components/accounts/ledger/SignInLedger.js

```diff
@@ -47,6 +48,7 @@ export function SignInLedger(props) {
     }
 
     const handleSignIn = async () => {
+        Mixpanel.track("IE-Ledger Click sign in button")
```

Comment:
I think It'd be useful to track both ledger sign in `start` and `finish` so we know how many succeed